wsbroadcastserver/connectionlimiter.go

```diff
@@ -8,9 +8,14 @@ import (
 	"sync"
 
 	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/metrics"
 	flag "github.com/spf13/pflag"
 )
 
+var (
+	clientsLimitedGauge = metrics.NewRegisteredGauge("arb/feed/clients/limited", nil)
```

Comment:
I think this should be a counter not a gauge, since it never goes down.

Reply:
I always mix these up, thanks